Tests/UnitsTests/UnitTests.swift

```diff
@@ -116,6 +147,11 @@ final class UnitTests: XCTestCase {
             "m/s^2"
         )
 
+        XCTAssertEqual(
+            (Unit.meter / Unit.second.pow(2.over(5))).symbol,
+            "m/s^(2|5)"
+        )
```

Comment:
Could we get some fractional exponent parser tests in this MR as well? It looks like we've only added tests that verify that we can go from Swift -> String, but not from String -> Swift, and I think we haven't updated the parser to support fractional exponent equations: https://github.com/CrownedPhoenix/Units/blob/87a14f4d211172678d5f0c70c19a4a71b2d6599a/Sources/Units/Parser.swift#L166